<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <title>software</title>
    <link href="../css/bootstrap.min.css" rel="stylesheet" media="screen">
    <link rel="stylesheet" type="text/css" href="../css/template_index.css">
</head>
<body>
<div>
    <div id="back"><img src="../img/bitl.png"></div>
    <div class="navbar">
        <div class="navbar-inner">

            <ul class="nav">
                <li><a href="../index.html">HOME</a></li>
                <li><a href="../introduction/introduction.html">Introduction</a></li>
                <li><a href="../professor/professor.html">Professor</a></li>
                <li><a href="../members/members.html">Members</a></li>
                <li><a href="../research/research_journal.html">Research</a></li>
                <li><a href="../education/education.html">Education</a></li>
                <li class="active"><a href="../sw/software.html">Software</a></li>
            </ul>
        </div>
    </div>
</div>
<div class="board">
    <div class="menu">
        <ul class="nav nav-pills nav-stacked">
            <li>
                <a href="./software.html">BIO SOFTWARE</a>
            </li>
            <li>
                <a href="./balancedSampling.html">balancedSampling</a>
            </li>
            <li>
                <a href="./f_interaction.html">f_interaction</a>
            </li>
            <li>
                <a href="./Hybrid_RFE.html">Hybrid RFE</a>
            </li>
        </ul>
    </div>

    <body bgcolor="#e1e1e1">
    <center>
        <h3> BIO SOFTWARE </h3>
        <p align="center">
            developed by BIO-IT Lab.<br>
            Dankook University, Korea<br>
        </p>
        <table width="750" height="500" border="1" rules="rows" frame="hsides" bgcolor="#e1e1e1">
            <tbody>
            <tr border="0">
                <td width="175" height="175">
                    <a href="AmRMR.html" target="_self">
                        <img src=".\img\AmRMR.JPG" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3">
                                <strong><a href="AmRMR.html" target="_self" style="color:blue; text-decoration:none">
                                    AmRMR</a>
                                    <br></strong>
                            </font> &nbsp; Minimum Redundancy - Maximum Relevance (mRMR) is one of well-known feature
                            selection algorithms that selects features by calculating redundancy and relevance between
                            features and class vector. mRMR uses mutual
                            information as a measure of redundancy and relevance. In this study, we proposed a method to
                            improve the performance of mRMR feature selection by using Pearson's correlation coefficient
                            as a redundancy measure and using R-value
                            as a relevance measure. We selected features by original mRMR and proposed method from
                            various datasets, and performed classification test. From the results, we confirmed that the
                            proposed method showed significant improvement
                            in classification accuracy in many cases.
                        </font>
                    </p>
                </td>
            </tr>

            <tr border="0">
                <td width="175" height="175">
                    <a href="R-value.html" target="_self">
                        <img src=".\img\r_value.JPG" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3">
                                <strong><a href="R-value.html" target="_self" style="color:blue; text-decoration:none">
                                    R-value</a>
                                    <br></strong>
                            </font> &nbsp;The quality of dataset has a profound effect on classification accuracy, and
                            there is a clear need for some method to evaluate this quality. R-value is a new dataset
                            evaluation method. This proposed method is
                            based on the ratio of overlapping areas among categories in a dataset. A high R-value for a
                            dataset indicates that the dataset contains wide overlapping areas among its categories
                            (classes), and classification accuracy on the
                            dataset may become low. We can use the R-value measure to understand the characteristics of
                            a dataset, the feature selection process, and the proper design of new classifiers.
                        </font>
                    </p>
                </td>
            </tr>
            <tr border="0">
                <td width="175" height="175">
                    <a href="RFS.html" target="_self">
                        <img src=".\img\RFS.PNG" width="150" height="150"></a>
                </td>

                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3">
                                <strong><a href="RFS.html" target="_self" style="color:blue; text-decoration:none">
                                    RFS </a>
                                    <br></strong>
                            </font>

                            &nbsp;We propose a new efficient feature selection method based on the <i>R-value</i>. The
                            original <i>R-value</i> was designed to evaluate the entire dataset, but we also found that
                            it could be applied to the feature selection
                            task using the modified <i>R(D)</i>. The <i>R-value</i>-based feature selection (RFS) method
                            scores the overlapping areas of each feature in candidate features, and then selects
                            features that have low <i>R-value</i>. &nbsp;Proposed
                            idea is simple, but powerful for feature selection.

                        </font>
                    </p>
                </td>


            </tr>
            <tr border="0">
                <td width="175" height="175">
                    <a href="ConcaveHull.html" target="_self" style="text-decoration:none">
                        <img src=".\img\concaveHull.jpg" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="ConcaveHull.html" target="_self"
                                                      style="color:blue; text-decoration:none">Concave Hull</a>
                                <br></strong></font>
                            &nbsp;The convex hull indicates the boundary of the minical convex set containing a given
                            nonempty finite set of point in the plane. The concave hull approach is a more advanced
                            approach used to capture the exact shape of the surface of a dataset. It
                            can increase performance of accuracy in machin learning areas. Our new concavehull algorithm
                            is n-dimensional concave whereas previous researches suggest for 2-dimension datasets.
                            Additionaly our concaveness measure and graph
                            can use to abtain information of geometric boundary.<br>
                        </font>
                    </p>
                </td>
            </tr>
            <tr border="0">
                <td width="175" height="175">
                    <a href="UniPrimer.html" target="_self">
                        <img src=".\img\primer.jpg" width="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong>UniPrimer
                                <br></strong>
                            </font>
                            Primer design for comparative analysis of the primate genomes.<br>
                        </font>
                    </p>
                </td>
            </tr>
            <tr border="0">
                <td width="175" height="175">
                    <a href="DAMCMC.html" target="_self" style="text-decoration:none">
                        <img src=".\img\damc-mc.PNG" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="DAMCMC.html" target="_self"
                                                      style="color:blue; text-decoration:none">DAMC-MC</a>
                                <br></strong></font>
                            &nbsp;Classification is one of the paramount techniques in machine learning and
                            computational biology. Various successful classification schemes have been proposed for
                            datasets which have binary classes and a few features. If a dataset has multiple classes
                            and huge features like microarray data, classification accuracy may be low, even though
                            feature selections are applied to reduce the dimensions of the dataset. Here we introduce
                            our new classification algorithm called "DAMC-MC"
                            which stands for "Divide-and-Merge Classification for Multi-Class datasets".<br>
                        </font>
                    </p>
                </td>
            </tr>
            <tr border="0">
                <td width="175" height="175">
                    <a href="SpinalCord.html" target="_self" style="text-decoration:none">
                        <img src=".\img\spinalcord.jpg" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="SpinalCord.html" target="_self"
                                                      style="color:blue; text-decoration:none">Spinal Cord</a>
                                <br></strong></font>


                            <br>
                        </font>
                    </p>
                </td>
            </tr>

            <tr border="0">
                <td width="175" height="175">
                    <a href="CBFS.html" target="_self" style="text-decoration:none">
                        <img src=".\img\cbfs_001.bmp" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="CBFS.html" target="_self"
                                                      style="color:blue; text-decoration:none">CBFS</a>
                                <br></strong></font>
                            High performance feature selection algorithm based on feature clearness<br>
                        </font>
                    </p>
                </td>
            </tr>


            <tr border="0">
                <td width="175" height="175">
                    <a href="AGM.html" target="_self" style="text-decoration:none">
                        <img src=".\img\agm.PNG" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="AGM.html" target="_self"
                                                      style="color:blue; text-decoration:none">AGM</a>
                                <br></strong></font>
                            &nbsp;We are suggesting a new method AGM (artificial gene making) to improve classification
                            accuracy. The role of artificial gene is to leave space among different classes of gene
                            selection result. Advantage of artificial gene is to reduce ambiguous or
                            congested areas among classes, which leads to improved classification accuracy.
                            <br>
                        </font>
                    </p>
                </td>
            </tr>


            <tr border="0">
                <td width="175" height="175">
                    <a href="boostMDR.html" target="_self" style="text-decoration:none">
                        <img src=".\img\boostMDR.png" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="boostMDR.html" target="_self"
                                                      style="color:blue; text-decoration:none">boostMDR</a>
                                <br></strong></font>
                            &nbsp;Boosting method that reduces the execution time of multifactor dimensionality
                            reduction by using pre-evaluation measurements to remove gene sets that have low interaction
                            before applying the reduction to the remaining sets.
                            <br>
                        </font>
                    </p>
                </td>
            </tr>

            <tr border="0">
                <td width="175" height="175">
                    <a href="postDiscretization.html" target="_self" style="text-decoration:none">
                        <img src="" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="postDiscretization.html" target="_self"
                                                      style="color:blue; text-decoration:none">postDiscretization</a>
                                <br></strong></font>

                            <br>
                        </font>
                    </p>
                </td>
            </tr>

            <tr border="0">
                <td width="175" height="175">
                    <a href="haploFinder.html" target="_self" style="text-decoration:none">
                        <img src="" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="haploFinder.html" target="_self"
                                                      style="color:blue; text-decoration:none">haploFinder</a>
                                <br></strong></font>

                            <br>
                        </font>
                    </p>
                </td>
            </tr>

            <tr border="0">
                <td width="175" height="175">
                    <a href="FindBiomarker.html" target="_self" style="text-decoration:none">
                        <img src=".\img\biomarker.png" width="150" height="150"></a>
                </td>
                <td align="left">
                    <p>
                        <font size="2" face="Verdana">
                            <font size="3"><strong><a href="FindBiomarker.html" target="_self"
                                                      style="color:blue; text-decoration:none">Find Biomarker</a>
                                <br></strong></font>
                            &nbsp;Data mining approach for finding biomarker genes based on microarray dataset
                            <br>
                        </font>
                    </p>
                </td>
            </tr>

            </tbody>
        </table>


        <br><br><br><br>

    </center>


    </body>
</div>
</body>
</html>